src/Compilers/Core/Portable/Operations/OperationKind.cs

```diff
@@ -141,8 +141,6 @@ public enum OperationKind
         AddressOfExpression = 0x203,
         /// <summary>Indicates an <see cref="IPointerIndirectionReferenceExpression"/>.</summary>
         PointerIndirectionReferenceExpression = 0x204,
-        /// <summary>Indicates an <see cref="IUnboundLambdaExpression"/>.</summary>
-        UnboundLambdaExpression = 0x205,
```

Comment:
@mavasani @dotnet/analyzer-ioperation, I think I prefer to leave this open for future use. Any objections?